components/tidb_query/src/rpn_expr/impl_math.rs

```diff
@@ -304,6 +304,117 @@ fn cot(arg: &Option<Real>) -> Result<Option<Real>> {
     }
 }
 
+// Copy from builtin_math.rs
+fn format_radix(mut x: u64, radix: u32) -> String {
+    let mut r = vec![];
+    loop {
+        let m = x % u64::from(radix);
+        x /= u64::from(radix);
+        r.push(
+            std::char::from_digit(m as u32, radix)
+                .unwrap()
+                .to_ascii_uppercase(),
+        );
+        if x == 0 {
+            break;
+        }
+    }
+    r.iter().rev().collect::<String>()
+}
+
+#[inline]
+#[rpn_fn]
+pub fn conv(
+    n: &Option<Bytes>,
+    from_base: &Option<Int>,
+    to_base: &Option<Int>,
+) -> Result<Option<Bytes>> {
+    #[derive(Copy, Clone)]
+    struct IntWithSign(u64, bool);
+
+    fn sep_neg_sign(num: Int) -> IntWithSign {
+        if num > 0 {
+            IntWithSign(num as u64, false)
```

Comment:
Could we simplify this code to: `IntWithSign(num.wrapping_abs() as u64, num > 0)`

Reply:
Done, and thanks for your advice.